doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -435,7 +435,7 @@ Bluetooth samples
   * The :ref:`ble_radio_notification_conn_cb` sample demonstrating how to use the :ref:`ug_radio_notification_conn_cb` feature.
   * The :ref:`bluetooth_conn_time_synchronization` sample demonstrating microsecond-accurate synchronization of connections that are happening over Bluetooth® Low Energy Asynchronous Connection-oriented Logical transport (ACL).
   * The :ref:`ble_subrating` sample that showcases the effect of the LE Connection Subrating feature on the duty cycle of a connection.
-  * The :ref:`broadcast_configuration_tool` sample that implements the :ref:`BIS gateway mode <nrf53_audio_app_overview>` and may act as an `Auracast™`_ broadcaster if you are using a preset compatible with Auracast.
+  * The :ref:`nrf_auraconfig` sample that implements the :ref:`BIS gateway mode <nrf53_audio_app_overview>` and may act as an `Auracast™`_ broadcaster if you are using a preset compatible with Auracast.
```

Comment:
This updated is added here: https://github.com/nrfconnect/sdk-nrf/pull/18300/files#diff-8e4512ec32b98eb4e48746453af3d0e501545e9cc009dc1f81611b29b87b2ab3. Please remove this file from this PR.

Reply:
Done

Reply:
@divipillai I now got: /home/runner/work/sdk-nrf/sdk-nrf/ncs/nrf/doc/_build/nrf/src/releases_and_maturity/releases/release-notes-changelog.rst:438: WARNING: undefined label: 'broadcast_configuration_tool'. And looking at your link it is to this PR. I might have misunderstood something